account_regularization/models/account.py

```diff
@@ -0,0 +1,168 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (c) 2008 Zikzakmedia S.L. (http://zikzakmedia.com)
+#    All Rights Reserved.
+#                       Jordi Esteve <jesteve@zikzakmedia.com>
+#    Copyright (c) 2008 Acysos SL. All Rights Reserved.
+#    $Id$
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, api, fields
+import time
+
+
+class AccountAccount(models.Model):
+    _name = 'account.account'
```

Comment:
Still a lot of things to learn.
Thanks  for the review!!!

2015-06-26 19:56 GMT+02:00 Pedro M. Baeza <notifications@github.com>:

> In account_regularization/models/account.py
> <https://github.com/OCA/account-financial-tools/pull/247#discussion_r33379171>
> :
>
> > +#    This program is distributed in the hope that it will be useful,
> > +#    but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > +#    GNU General Public License for more details.
> > +#
> > +#    You should have received a copy of the GNU General Public License
> > +#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +##############################################################################
> > +
> > +from openerp import models, api, fields
> > +import time
> > +
> > +
> > +class AccountAccount(models.Model):
> > +    _name = 'account.account'
>
> You don't need to repeat this variable
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/account-financial-tools/pull/247/files#r33379171>.
>